pkg/streamingpromql/functions.go

```diff
@@ -43,17 +43,17 @@ func SingleInputVectorFunctionOperatorFactory(name string, metadataFunc function
 	}
 }
 
-// TransformationFunctionOperatorFactory creates an InstantVectorFunctionOperatorFactory for functions
+// SingleInputVectorTransformationFunctionOperatorFactory creates an InstantVectorFunctionOperatorFactory for functions
```

Comment:
This name is quite wordy - how about `InstantVectorTransformationFunctionOperatorFactory`?

Any function that doesn't take in a single argument doesn't qualify as a simple transformation function in my mind, so I think we can drop that part of the name.